Lib/rlcompleter.py

```diff
@@ -185,6 +191,12 @@ def attr_matches(self, text):
         matches.sort()
         return matches
 
+_re_ignorecase_flags = 0
+def set_case_insensitive(option):
```

Comment:
I find it weird that the way to be case sensitive is to have a double negation:

```python
set_case_insensitive(False)
```

can we change it to:

```python
set_case_sensitive(True)
```

?

Reply:
I like your comment. If you see the manual page of readline() it is also named to have double negation: 
`completion-ignore-case (Off)`
`       If set to On, readline performs filename matching and completion in a case-insensitive fashion.`

Instead, I would like to name it more similar to readline() as set_ignore_case().